src/App.css

```diff
@@ -1,22 +1,40 @@
-.App {
-  text-align: center;
+.Buttons {
+  display: flex;
+  justify-content: space-around;
+  background-color: #282c34;
 }
 
-.App-logo {
-  height: 40vmin;
+.botton {
+  text-decoration: none;
+  font-size: 20px;
+  color: #fefefe;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-left: 40px;
+  padding-right: 40px;
+  background-color: #09d3ac;
+  border-color: #d8d8d8;
+  border-width: 3px;
+  border-style: solid;
+  border-radius: 35px;
+  margin-right: 10px;
+  text-decoration: none;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.no-decoration {
```

Comment:
Did you discuss a naming convention in order to maintain consistency in the  CSS classes? 
Take a look to this [BEM](http://getbem.com/l)